geoutils/georaster.py

```diff
@@ -40,24 +40,25 @@ class Raster(object):
     filename = None
     matches_disk = None
 
-    def __init__(self, filename, attrs=None, load_data=True, bands=None,
-                 masked=True, as_memfile=False):
-
+    def __init__(self, filename, bands=None, load_data=True, downsampl=1,
+                 masked=True, attrs=None, as_memfile=False):
         """
         Load a rasterio-supported dataset, given a filename.
 
         :param filename: The filename of the dataset.
         :type filename: str
-        :param attrs: Additional attributes from rasterio's DataReader class to add to the Raster object.
-            Default list is ['bounds', 'count', 'crs', 'dataset_mask', 'driver', 'dtypes', 'height', 'indexes',
-            'name', 'nodata', 'res', 'shape', 'transform', 'width'] - if no attrs are specified, these will be added.
-        :type attrs: list of strings
-        :param load_data: Load the raster data into the object. Default is True.
-        :type load_data: bool
         :param bands: The band(s) to load into the object. Default is to load all bands.
         :type bands: int, or list of ints
+        :param load_data: Load the raster data into the object. Default is True.
```

Comment:
took me 20s to realize you switched this to be consistent with order of  parameters!